x/staking/module.go

```diff
@@ -212,8 +235,14 @@ type stakingOutputs struct {
 }
 
 func provideModule(in stakingInputs) stakingOutputs {
-	k := keeper.NewKeeper(in.Cdc, in.Key, in.AccountKeeper, in.BankKeeper, in.Subspace)
-	m := NewAppModule(in.Cdc, k, in.AccountKeeper, in.BankKeeper)
+	k := keeper.NewKeeper(
+		in.Cdc,
+		in.Key,
+		in.AccountKeeper,
+		in.BankKeeper,
+		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
```

Comment:
We've changed it to have it non hard-coded anymore: c.f. https://github.com/cosmos/cosmos-sdk/pull/12660